instagram-page-react/src/molecules/header/atoms/headerActions/headerActions.constants.js

```diff
@@ -0,0 +1,14 @@
+const headerActionsConstants = [
```

Comment:
1. headerActionsConstants  <> HEADER_ACTION_CONFIGS
2. Move to constants/headerActions.configs.js
3. We should define each config separately, LOGIN_ACTION_CONFIG, SIGNUP_ACTION_CONFIG